packages/eslint-plugin-pf-codemods/lib/rules/nav-list-variant.js

```diff
@@ -28,21 +35,35 @@ module.exports = {
             if (variantAttr.value !== null) {
               context.report({
                 node,
-                message: `variant has been removed from ${node.openingElement.name.name}, use <${navImportName} ${variantVal}> instead`,
+                messageId: "removeNavListVariant",
+                data: {
+                  navListName: node.openingElement.name.name,
+                  navName: navImportName,
+                  variantVal: variantVal
+                },
                 fix(fixer) {
-                  const fixes = [fixer.replaceText(variantAttr, '')];
+                  const fixes = [];
                   if (hasNavParent) {
```

Comment:
need to add checks if user used `simple` variant before, since that doesn't map over to the new Nav variant prop

Reply:
Done

Reply:
Did you push @ddonahue007 ?

Reply:
I thought i pushed, I just did another push see what happens